doc/development/source/orange-demo/setup.py

```diff
@@ -5,5 +5,5 @@
       package_data={"orangedemo": ["icons/*.svg"]},
       classifiers=["Example :: Invalid"],
       # Declare orangedemo package to contain widgets for the "Demo" category
-      entry_points={"orange.widgets": ("Demo = orangedemo")},
-      )
\ No newline at end of file
+      entry_points={"orange.widgets": "Demo = orangedemo"},
```

Comment:
In Python, these parentheses should make no difference:
```py
assert 'Demo = orangedemo' == ("Demo = orangedemo")
```
It's only a tuple when the item list (of a single item) ends with a comma: `('whatever',)`

The reason for it not working before must have been somewhere else.